task_6.py

```diff
@@ -0,0 +1,9 @@
+# Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с номером. Счастливым билетом называют такой билет с шестизначным номером, 
+# где сумма первых трех цифр равна сумме последних трех. Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, 
+# которая проверяет счастливость билета.
+
+# Запрашиваем ввод номера билета
+number = input("Введите шестизначный номер билета: ")
+
+# Проверяем и выводим информацию о "счасливости" билета
+print("Билет счастливый! Удача на твоей стороне!" if (int(number[0]) + int(number[1]) + int(number[2]) == int(number[3]) + int(number[4]) + int(number[5])) else "Увы! Билет - несчастливый! Повезет в другой раз!")
```

Comment:
Артур, слишком длинная строка для решения в одну строку. Да и внешние скобки в условии тут все равно не нужны.
```suggestion
print("Билет счастливый! Удача на твоей стороне!" if int(number[0]) + int(number[1]) + int(number[2]) == int(number[3]) + int(number[4]) + int(number[5]) else "Увы! Билет - несчастливый! Повезет в другой раз!")
```